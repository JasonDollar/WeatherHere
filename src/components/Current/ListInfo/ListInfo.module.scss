.Current {
  display: flex;
  flex-wrap: wrap;
  margin: 2.5rem 0;
  list-style: none;

}

.element {
  font-size: var(--normal-text-font-size);

  margin: 0 auto;
  flex: 0 0 95%;
  padding: .5rem 2rem;
  display: flex;
  justify-content: space-between;

  & span {
    font-weight: 300;
    padding-left: 2px;
  }


  @media only screen and (min-width: 630px) {
    & {
      margin: 0;
      flex: 0 0 50%;
      font-size: var(--medium-text-font-size);
    }
  }

  @media only screen and (min-width: 960px) {
    & {
      flex: 0 0 33.3333%;
    }
  }
}