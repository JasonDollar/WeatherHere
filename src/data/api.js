const DARK_API = 'a077c3a2c30623e41ef4a1cafbec622d'
const DARK_URL = 'https://api.darksky.net/forecast/'

export { DARK_API, DARK_URL }  

