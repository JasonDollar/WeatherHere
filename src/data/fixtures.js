import weatherData from './random'
import languageText from './text-locale' 

export const appTestData = {
  weather: weatherData,
  units: 'si',
  language: 'pl',
  text: languageText,
  timezone: 'Europe/Warsaw',
  locationShortName: 'Plock',
  
}