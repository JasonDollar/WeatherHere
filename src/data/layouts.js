const layouts = {
  pink: {
    id: 'pink',
    name: 'Pink',
    primary: '#dd0055',
    primaryOffset: '#f5689e',
    mobileMenu: '#dd0055',
    antiGraph: '#8884d8',
    secondaryGraph: '#0c1052',
    background: '#fffcfc',
    background2: '#fffcfc',
    backgroundMobile: '#ddd',
    text: '#222',
    textSecondary: '#eee',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#bbb',
    grey3: '#eae9e9',
    greyGraph: '#676767',
  },
  pinkDark: {
    id: 'pinkDark',
    name: 'Pink Dark',
    primary: '#dd0055',
    primaryOffset: '#dd0055',
    mobileMenu: '#f5689e',
    antiGraph: '#8884d8',
    secondaryGraph: '#dd0055',
    background: '#272424',
    background2: '#fcfcfc',
    backgroundMobile: '#333',
    text: '#efeee7',
    textSecondary: '#ddd',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#999',
    grey3: '#333',
    greyGraph: '#acacac',
  },
  pumpkin: {
    id: 'pumpkin',
    name: 'Pumpkin Dark',
    primary: '#f48042',
    primaryOffset: '#ef915f',
    mobileMenu: '#f48042',
    antiGraph: '#50c25f',
    secondaryGraph: '#f48042',
    background: '#262424',
    background2: '#fcfcfc',
    backgroundMobile: '#333',
    text: '#eee',
    textSecondary: '#ddd',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#999',
    grey3: '#363333',
    greyGraph: '#acacac',
  },
  evening: {
    id: 'evening',
    name: 'Evening Dark',
    primary: '#b92b27',
    primaryOffset: '#d84f4b',
    mobileMenu: '#d84f4b',
    antiGraph: '#1565C0',
    secondaryGraph: '#d84f4b',
    background: '#282424',
    background2: '#fcfcfc',
    backgroundMobile: '#333',
    text: '#eeeef8',
    textSecondary: '#ddd',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#999',
    grey3: '#363333',
    greyGraph: '#acacac',
  },
  sky: {
    id: 'sky',
    name: 'Cool Sky',
    primary: '#2980B9',
    primaryOffset: '#6DD5FA',
    mobileMenu: '#2980B9',
    antiGraph: '#756ef0',
    secondaryGraph: '#2980B9',
    background: '#fcfcfc',
    background2: '#fcfcfc',
    backgroundMobile: '#ddd',
    text: '#222',
    textSecondary: '#eee',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#bbb',
    grey3: '#e7e7ec',
    greyGraph: '#676767',
  },
  harvey: {
    id: 'harvey',
    name: 'Harvey',
    primary: '#2b574b',
    primaryOffset: '#75d1a6',
    mobileMenu: '#386e60',
    antiGraph: '#0daeca',
    secondaryGraph: '#2b574b',
    background: '#fcfcfc',
    background2: '#fcfcfc',
    backgroundMobile: '#ddd',
    text: '#222',
    textSecondary: '#eee',
    textTransparent: '#22222280',
    border: '#aaa',
    borderTransparent: '#aaaaaa55',
    boxShadow: '#222',
    grey1: '#888',
    grey2: '#bbb',
    grey3: '#e7eae7',
    greyGraph: '#676767',
  },


}

export default layouts



