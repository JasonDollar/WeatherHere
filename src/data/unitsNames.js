export const unitsNames = {
  si: {
    id: 'si',
    distance: 'km',
    precip: 'mm/h',
    precipAcc: 'cm',
    temperature: '째C',
    wind: 'm/s',
    pressure: 'hPa',
  },
  us: {
    id: 'us',
    distance: 'mi',
    precip: 'in/h',
    precipAcc: 'in',
    temperature: '째F',
    wind: 'mi/h',
    pressure: 'hPa',
  },
  uk2: {
    id: 'uk2',
    distance: 'mi',
    precip: 'mm/h',
    precipAcc: 'cm',
    temperature: '째C',
    wind: 'mi/h',
    pressure: 'hPa',
  },
  ca: {
    id: 'ca',
    distance: 'km',
    precip: 'mm/h',
    precipAcc: 'cm',
    temperature: '째C',
    wind: 'km/h',
    pressure: 'hPa',
  },
}