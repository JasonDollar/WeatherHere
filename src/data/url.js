const DARK_URL = 'https://api.darksky.net/forecast/'

export { DARK_URL }