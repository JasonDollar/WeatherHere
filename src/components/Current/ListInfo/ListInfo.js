import React from 'react'

import { formatNumber } from '../../../data/utils'

import classes from './ListInfo.module.scss'

const ListInfo = ({ currentText, currently, moonphase }) => (
  <div className={classes.Current}>
  
    <div className={classes.element}>
      <div>{currentText.appTemp}</div>
      <div>
        {formatNumber(currently.apparentTemperature)}
        <span>°C</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.dewPoint}</div>
      <div>
        {formatNumber(currently.dewPoint)}
        <span>°C</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.pressure}</div>
      <div>
        {formatNumber(currently.pressure)} 
        <span>hPa</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.windGust}</div>
      <div>
        {formatNumber(currently.windGust)} 
        <span>m/s</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.precipInt}</div>
      <div>
        {formatNumber(currently.precipIntensity)} 
        <span>mm/h</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.visibility}</div>
      <div>
        {formatNumber(currently.visibility)}
        <span>km</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.uvIndex}</div>
      <div>{formatNumber(currently.uvIndex)}</div>
    </div>
    <div className={classes.element}>
      <div>{currentText.moonphase}</div>
      <div>
        {moonphase}
        <span>%</span>
      </div>
    </div>
    <div className={classes.element}>
      <div>{currentText.ozone}</div>
      <div>{formatNumber(currently.ozone)}</div>
    </div>

  </div>
)

export default ListInfo
