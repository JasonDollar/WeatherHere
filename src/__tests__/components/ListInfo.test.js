import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { appTestData as data } from '../../data/fixtures'
import ListInfo from '../../components/Current/ListInfo/ListInfo'

let currentText 
let currently 
let moonphase 
let units
let wrapper

beforeAll(() => {
  currentText = data.text.pl.current
  currently = data.weather.currently
  moonphase = 0.5
  units = data.units.si.id
  wrapper = shallow(
    <ListInfo 
      currentText={currentText}
      currently={currently}
      moonphase={moonphase}
      units={units}
    />,
  )
})

describe('ListInfo component test', () => {
  test('Matches snapshot', () => {
    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})