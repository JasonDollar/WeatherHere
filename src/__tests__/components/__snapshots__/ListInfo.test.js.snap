// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListInfo comp. test ListInfo match to Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ListInfo
    currentText={
      Object {
        "appTemp": "Temperatura odczuwalna",
        "cloudCov": "Zachmurzenie",
        "date": "Data",
        "dewPoint": "Punkt rosy",
        "humidity": "Wilgotność",
        "icon": "Ikona pogody",
        "location": "Nieznana lokacja",
        "moon": Array [
          "Nów",
          "Wschodzący sierp",
          "Pierwsza kwadra",
          "Wschodzący księżyc",
          "Pełnia",
          "Ubywający księżyc",
          "Trzecia kwadra",
          "Ubywający sierp",
        ],
        "moonphase": "Faza księżyca",
        "ozone": "Ozon",
        "precipInt": "Intensywność opadu",
        "precipProb": "Opady",
        "precipType": "Rodzaj opadu",
        "pressure": "Ciśnienie",
        "sun": "Słońce",
        "sunrise": "Wschód słońca",
        "sunset": "Zachód słońca",
        "today": "Dziś",
        "uvIndex": "Indeks UV",
        "visibility": "Widoczność",
        "wind": "Wiatr",
        "windGust": "Wiatr w porywach",
      }
    }
    currently={
      Object {
        "apparentTemperature": 6.53,
        "cloudCover": 0.92,
        "dewPoint": 8.45,
        "humidity": 0.94,
        "icon": "partly-cloudy-night",
        "ozone": 290.56,
        "precipIntensity": 0.0076,
        "precipProbability": 0.01,
        "precipType": "rain",
        "pressure": 1009.97,
        "summary": "Średnie zachmurzenie",
        "temperature": 9.38,
        "time": 1540834485,
        "uvIndex": 0,
        "visibility": 16.09,
        "windBearing": 99,
        "windGust": 10.76,
        "windSpeed": 5.76,
      }
    }
    moonphase={0.5}
    units="si"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="element"
        >
          <div>
            Temperatura odczuwalna
          </div>
          <div>
            6.5
            <span>
              °C
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Punkt rosy
          </div>
          <div>
            8.4
            <span>
              °C
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Ciśnienie
          </div>
          <div>
            1009.9
            <span>
              hPa
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Wiatr w porywach
          </div>
          <div>
            10.7
            <span>
              m/s
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Intensywność opadu
          </div>
          <div>
            0
            <span>
              mm/h
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Widoczność
          </div>
          <div>
            16
            <span>
              km
            </span>
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Indeks UV
          </div>
          <div>
            0
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Faza księżyca
          </div>
          <div>
            Pełnia
          </div>
        </div>,
        <div
          className="element"
        >
          <div>
            Ozon
          </div>
          <div>
            290.5
            <span>
              DU
            </span>
          </div>
        </div>,
      ],
      "className": "Current",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Temperatura odczuwalna
            </div>,
            <div>
              6.5
              <span>
                °C
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Temperatura odczuwalna",
            },
            "ref": null,
            "rendered": "Temperatura odczuwalna",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                6.5,
                <span>
                  °C
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              6.5,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "°C",
                },
                "ref": null,
                "rendered": "°C",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Punkt rosy
            </div>,
            <div>
              8.4
              <span>
                °C
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Punkt rosy",
            },
            "ref": null,
            "rendered": "Punkt rosy",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                8.4,
                <span>
                  °C
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              8.4,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "°C",
                },
                "ref": null,
                "rendered": "°C",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Ciśnienie
            </div>,
            <div>
              1009.9
              <span>
                hPa
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ciśnienie",
            },
            "ref": null,
            "rendered": "Ciśnienie",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                1009.9,
                <span>
                  hPa
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              1009.9,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "hPa",
                },
                "ref": null,
                "rendered": "hPa",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Wiatr w porywach
            </div>,
            <div>
              10.7
              <span>
                m/s
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Wiatr w porywach",
            },
            "ref": null,
            "rendered": "Wiatr w porywach",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                10.7,
                <span>
                  m/s
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              10.7,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "m/s",
                },
                "ref": null,
                "rendered": "m/s",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Intensywność opadu
            </div>,
            <div>
              0
              <span>
                mm/h
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Intensywność opadu",
            },
            "ref": null,
            "rendered": "Intensywność opadu",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                0,
                <span>
                  mm/h
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              0,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "mm/h",
                },
                "ref": null,
                "rendered": "mm/h",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Widoczność
            </div>,
            <div>
              16
              <span>
                km
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Widoczność",
            },
            "ref": null,
            "rendered": "Widoczność",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                16,
                <span>
                  km
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              16,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "km",
                },
                "ref": null,
                "rendered": "km",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Indeks UV
            </div>,
            <div>
              0
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Indeks UV",
            },
            "ref": null,
            "rendered": "Indeks UV",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": 0,
            },
            "ref": null,
            "rendered": 0,
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Faza księżyca
            </div>,
            <div>
              Pełnia
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Faza księżyca",
            },
            "ref": null,
            "rendered": "Faza księżyca",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Pełnia",
            },
            "ref": null,
            "rendered": "Pełnia",
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Ozon
            </div>,
            <div>
              290.5
              <span>
                DU
              </span>
            </div>,
          ],
          "className": "element",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Ozon",
            },
            "ref": null,
            "rendered": "Ozon",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                290.5,
                <span>
                  DU
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              290.5,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "DU",
                },
                "ref": null,
                "rendered": "DU",
                "type": "span",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="element"
          >
            <div>
              Temperatura odczuwalna
            </div>
            <div>
              6.5
              <span>
                °C
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Punkt rosy
            </div>
            <div>
              8.4
              <span>
                °C
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Ciśnienie
            </div>
            <div>
              1009.9
              <span>
                hPa
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Wiatr w porywach
            </div>
            <div>
              10.7
              <span>
                m/s
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Intensywność opadu
            </div>
            <div>
              0
              <span>
                mm/h
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Widoczność
            </div>
            <div>
              16
              <span>
                km
              </span>
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Indeks UV
            </div>
            <div>
              0
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Faza księżyca
            </div>
            <div>
              Pełnia
            </div>
          </div>,
          <div
            className="element"
          >
            <div>
              Ozon
            </div>
            <div>
              290.5
              <span>
                DU
              </span>
            </div>
          </div>,
        ],
        "className": "Current",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Temperatura odczuwalna
              </div>,
              <div>
                6.5
                <span>
                  °C
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Temperatura odczuwalna",
              },
              "ref": null,
              "rendered": "Temperatura odczuwalna",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  6.5,
                  <span>
                    °C
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                6.5,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "°C",
                  },
                  "ref": null,
                  "rendered": "°C",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Punkt rosy
              </div>,
              <div>
                8.4
                <span>
                  °C
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Punkt rosy",
              },
              "ref": null,
              "rendered": "Punkt rosy",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  8.4,
                  <span>
                    °C
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                8.4,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "°C",
                  },
                  "ref": null,
                  "rendered": "°C",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Ciśnienie
              </div>,
              <div>
                1009.9
                <span>
                  hPa
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ciśnienie",
              },
              "ref": null,
              "rendered": "Ciśnienie",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  1009.9,
                  <span>
                    hPa
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                1009.9,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "hPa",
                  },
                  "ref": null,
                  "rendered": "hPa",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Wiatr w porywach
              </div>,
              <div>
                10.7
                <span>
                  m/s
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Wiatr w porywach",
              },
              "ref": null,
              "rendered": "Wiatr w porywach",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  10.7,
                  <span>
                    m/s
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                10.7,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "m/s",
                  },
                  "ref": null,
                  "rendered": "m/s",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Intensywność opadu
              </div>,
              <div>
                0
                <span>
                  mm/h
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Intensywność opadu",
              },
              "ref": null,
              "rendered": "Intensywność opadu",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  0,
                  <span>
                    mm/h
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                0,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "mm/h",
                  },
                  "ref": null,
                  "rendered": "mm/h",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Widoczność
              </div>,
              <div>
                16
                <span>
                  km
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Widoczność",
              },
              "ref": null,
              "rendered": "Widoczność",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  16,
                  <span>
                    km
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                16,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "km",
                  },
                  "ref": null,
                  "rendered": "km",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Indeks UV
              </div>,
              <div>
                0
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Indeks UV",
              },
              "ref": null,
              "rendered": "Indeks UV",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": 0,
              },
              "ref": null,
              "rendered": 0,
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Faza księżyca
              </div>,
              <div>
                Pełnia
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Faza księżyca",
              },
              "ref": null,
              "rendered": "Faza księżyca",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Pełnia",
              },
              "ref": null,
              "rendered": "Pełnia",
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Ozon
              </div>,
              <div>
                290.5
                <span>
                  DU
                </span>
              </div>,
            ],
            "className": "element",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ozon",
              },
              "ref": null,
              "rendered": "Ozon",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  290.5,
                  <span>
                    DU
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                290.5,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "DU",
                  },
                  "ref": null,
                  "rendered": "DU",
                  "type": "span",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
